build_tools/azure/install.cmd

```diff
@@ -25,7 +25,7 @@ IF "%PYTHON_ARCH%"=="64" (
     pip install numpy scipy cython pytest wheel pillow joblib
 )
 if "%COVERAGE%" == "true" (
-    pip install coverage codecov pytest-cov
+    pip install coverage==4.5.3 codecov pytest-cov
```

Comment:
When we do such pinning we should always add a comment to explain why it's needed so that we know when it's safe to remove the pinning in the future.

Reply:
Ah yes, the comment was added in https://github.com/scikit-learn/scikit-learn/pull/15912